packages/plugin-loansresearch-api/src/models/definitions/loansResearch.ts

```
@@ -0,0 +1,90 @@
+import { Schema, Document } from 'mongoose';
+import { field, schemaWrapper } from './utils';
+import { IAttachment } from '@erxes/api-utils/src/types';
+
+export interface IIncome {
+  _id: string;
+  incomeType: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)